javascript/components/MapView.js

```diff
@@ -257,9 +258,14 @@ class MapView extends NativeBridgeComponent {
   }
 
   componentDidMount() {
+    this._mounted = true;
     this._setHandledMapChangedEvents(this.props);
   }
 
+  componentWillUnmount() {
+    this._mounted = false;
```

Comment:
I guess the reason `_onRegionDidChange` can be called on an unmounted component is because of `debounce`. I'd consider just cancelling the debounced callbacks.
```suggestion
    this._onDebouncedRegionWillChange.cancel();
    this._onDebouncedRegionDidChange.cancel();
```

Reply:
I updated the pull. Using `.cancel()` was a way better proposal, thanks for that!